Package@swift-6.swift

```diff
@@ -798,6 +798,17 @@ extension Target {
       swiftSettings: [.swiftLanguageVersion(.v5)]
     )
   }
+
+  static var grpcHealth: Target {
+    .target(
+      name: "GRPCHealth",
+      dependencies: [
+        .grpcCore,
+        .grpcProtobuf
+      ],
+      path: "Sources/Services/Health"
+    )
```

Comment:
Can you add: `swiftSettings: [.swiftLanguageVersion(.v6)]` here? That will enable the Swift 6 language mode for this module.